client/public/index.html

```diff
@@ -14,6 +14,7 @@
     <link
       rel="stylesheet"
       type="text/css"
+      href="https://fonts.googleapis.com/css?family=Open+Sans"
       href="https://fonts.googleapis.com/css?family=Roboto:300,400,500,700|Material+Icons"
```

Comment:
Only have one href and should be:
`https://fonts.googleapis.com/css?family=Open+Sans|Material+Icons`

Reply:
OK